cpp/example_code/cloudtrail/create_trail.cpp

```diff
@@ -7,44 +7,83 @@
 #include <aws/cloudtrail/model/CreateTrailRequest.h>
 #include <aws/cloudtrail/model/CreateTrailResult.h>
 #include <iostream>
+#include "cloudtrail_samples.h"
 
 /**
- * Creates a cloud trail on command line input
+ * Before running this C++ code example, set up your development environment, including your credentials.
+ *
+ * For more information, see the following documentation topic:
+ *
+ * https://docs.aws.amazon.com/sdk-for-cpp/v1/developer-guide/getting-started.html
+ *
+ * Purpose
+ *
+ * Demonstrates using the AWS SDK for C++ to list the objects in an S3 bucket.
```

Comment:
This comment seems incorrect for this function.

Reply:
fixed